lang-java-reach/src/main/java/com/sap/psr/vulas/cg/Callgraph.java

```diff
@@ -263,29 +211,11 @@ public NodeMetaInformation getInformationForId(Integer id){
 	 */
 	public int getEdgeCount() { return this.edgeCount; }
 
-	/**
-	 * Returns the {@link JavaPackageId} of a given {@link ConstructId}.
-	 * 
-	 * @param _cid
-	 * @return
-	 */
-	public static JavaPackageId getPackageId (ConstructId _cid) {
```

Comment:
Why is this not needed anylonger?

Reply:
This one was not used any more, it has been replaced some time ago by a corresponding method `JavaId.getJavaPackageId()`.